tests/test_utils.py

```diff
@@ -27,3 +31,29 @@ def test_nb_nll_moments(mu, alpha):
     assert np.abs(diff) < 0.2 * deviation
     error_var = np.abs(sample.var() - var_th) / var_th
     assert error_var < 1 / np.sqrt(n_montecarlo)
+
+
+# Test data loading from outside the package (e.g. on RTF)
```

Comment:
```suggestion
# Test data loading from outside the package (e.g. on RTF)
@pytest.mark.parametrize("return_value", [True, False])
```

why not to directly test for both values? and set return_value to the that parameter

Reply:
Can this work in conjunction with `mocking`?

Reply:
I don't see why not. but test it ;-)

Reply:
Done! (in [fe6d79f](https://github.com/owkin/PyDESeq2/pull/83/commits/fe6d79fc2a89ee34bd8524cdc87fc11cd610e25d))